stackdriver-core/lib/stackdriver/core/async_actor.rb

```diff
@@ -199,6 +199,30 @@ def async_stop!
       end
 
       ##
+      # Block current thread until the async job is fully stopped.
```

Comment:
Shouldn't this still be labeled `@private`? You want to be able to call this method from other objects as part of the implementation, but you don't want this method to be considered part of the public API, right?